saleor/graphql/app/types.py

```diff
@@ -85,4 +125,24 @@ def __resolve_reference(root, _info, **_kwargs):
 
     @staticmethod
     def resolve_webhooks(root: models.App, _info):
-        return root.webhooks.all()  # type: ignore
+        return root.webhooks.all()
+
+    @staticmethod
+    def resolve_access_token(root: models.App, info):
+        return resolve_access_token(info, root)
+
+
+class AppInstallation(CountableDjangoObjectType):
+    class Meta:
+        model = models.AppInstallation
+        description = "Represents ongoing installation of app."
+        interfaces = [graphene.relay.Node, Job]
+        permissions = (AppPermission.MANAGE_APPS,)
+        only_fields = [
+            "status",
+            "created_at",
+            "updated_at",
+            "app_name",
+            "manifest_url",
+            "message",
+        ]
```

Comment:
You don't have to provide fields that are specified in `Job` interface, I mean `status`, `creates_at`, `updated_at`. I see that `message` is the `Job` field so it could be added to the interface too.